index.js

```diff
@@ -0,0 +1,83 @@
+const hamburger = document.querySelector('#menu-button');
+const closeButton = document.querySelector('#close-menu');
+const mobileMenu = document.querySelector('.hidden');
+
+hamburger.addEventListener('click', () => {
+  mobileMenu.style = 'display: flex; z-index:0';
+  hamburger.style = 'display: none';
+  document.querySelector('body').style = 'overflow: hidden';
+  document.querySelector('.campaign').style = 'display: none';
+});
+
+closeButton.addEventListener('click', () => {
+  mobileMenu.style = 'display: none';
+  hamburger.style = 'display: block';
+  document.querySelector('body').style = 'overflow: scroll';
+  document.querySelector('.campaign').style = 'display: flex';
+  window.location.reload();
```

Comment:
- [ ] When one closes the mobile menu, the page shouldn't reload as this is an expensive process. Kindly remove the reload option.